lib/LedgerSMB/Database/Change.pm

```diff
@@ -155,8 +155,7 @@ Returns true if the current sha matches one that has been applied.
 sub is_applied {
     my ($self, $dbh) = @_;
     my $sha = $self->sha;
-    # Clone to isolate ourselves
-    my $sth = $dbh->clone({ AutoCommit => 1})->prepare(
+    my $sth = $dbh->prepare(
```

Comment:
Now the clone() is gone, but we're not cloning the dbh anyhere now... I think we should do the entire loading of the change under a cloned db handle, because we want to change the handle for the duration of the change application to `AutoCommit => 1` (or `0`, whatever the change warrants)

Reply:
No we don't need autocommit anymore, so I'm with your original questionning, why would a select need autocommit? It doesn't.